lib/index.js

```diff
@@ -1,13 +1,37 @@
 /*!
- * Copyright (c) 2018-2023 Digital Bazaar, Inc. All rights reserved.
+ * Copyright (c) 2023 Digital Bazaar, Inc. All rights reserved.
  */
-//import * as bedrock from '@bedrock/core';
-//import assert from 'assert-plus';
-//import {logger} from './logger.js';
+import * as bedrock from '@bedrock/core';
+import {agent} from '@bedrock/https-agent';
+import {httpClient} from '@digitalbazaar/http-client';
 
-// load config defaults
-import './config.js';
+const {config, util: {BedrockError}} = bedrock;
 
-/**
- * @module bedrock-turnstile
- */
+export async function verifyTurnstileToken({token, remoteip}) {
+  const cfg = config.turnstile;
+  const {SECRET_KEY, url} = cfg.authorization.turnstile;
```

Comment:
Should use assert-plus here (guessing the syntax below from memory on its use):

```suggestion
  const clientConfig = cfg.clients[client];
  assert(clientConfig, 'object', `config.turnstile.clients.${client}`);
  const {SECRET_KEY, url} = clientConfig;
```